### PR TITLE
Release Notes for Aug. 19 Release

### PR DESCRIPTION
**Pull request by huangjeff5**

### CODE REVIEW

File: delete-user-data/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## Version 0.1.10
+
+feature - added warsaw (europe-central2) location (#677)
```
Comment (kevinthecheung) [inline delete-user-data/CHANGELOG.md:3]: Same as 0.1.9?

Comment (huangjeff5) [reply delete-user-data/CHANGELOG.md]: Good catch - updated

File: firestore-translate-text/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+## Version 0.1.7
+
+feature - added warsaw (europe-central2) location (#677)
```
Comment (kevinthecheung) [inline firestore-translate-text/CHANGELOG.md:3]: Same

File: firestore-send-email/CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,11 @@
+## Version 0.1.11
+
+fixed - added fallback for finding email templates
+
+fixed - removed duplicate warsaw location
```
Comment (kevinthecheung) [inline firestore-send-email/CHANGELOG.md:5]:
```suggestion
fixed - removed duplicate Warsaw location
```

File: firestore-send-email/CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,11 @@
...
+
+feature - added warsaw (europe-central2) location (#677)
```
Comment (kevinthecheung) [inline firestore-send-email/CHANGELOG.md:7]: dupe?

File: firestore-bigquery-export/CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,7 @@
+## Version 0.1.17
+
+fixed - Improved import script
```
Comment (kevinthecheung) [inline firestore-bigquery-export/CHANGELOG.md:3]:
```suggestion
fixed - improved import script
```

File: delete-user-data/CHANGELOG.md [hunk 6]
```diff
@@ -1,3 +1,7 @@
+## Version 0.1.10
+
+feature - updated rtdb to be optional (#722)
```
Comment (kevinthecheung) [inline delete-user-data/CHANGELOG.md:3]: RTDB?

Comment (huangjeff5) [reply delete-user-data/CHANGELOG.md]: Fixed

File: firestore-send-email/CHANGELOG.md (hunk 4 above)
Comment (huangjeff5) [reply firestore-send-email/CHANGELOG.md]: Fixed thsi one

File: firestore-translate-text/CHANGELOG.md (hunk 2 above)
Comment (huangjeff5) [reply firestore-translate-text/CHANGELOG.md]: Realized we may not need a version update to this extension so reverting changes in this file.

**Review by kevinthecheung — APPROVED** [review]

**Pull request merged**